pontoon/sync/templates/sync/log_list.html

```diff
@@ -56,6 +59,11 @@ <h4>
               <td class="duration" title='{% if project.sync_end_time %}Latest End: {{ project.sync_end_time|date("Y-m-d H:i:s") }} UTC{% endif %}' data-sort="{{ format_sync_duration(project.sync_start_time, project.sync_end_time, ms=True) }}">
                 {{ format_sync_duration(project.sync_start_time, project.sync_end_time) }}
               </td>
+              {% if settings.MANUAL_SYNC and is_manager and not project.system_project %}
```

Comment:
It's the DB projects (not system projects) for which we should not show the Sync button. 

We should also hide that Sync button for those in the Project Admin.